File: Madrassaty.Back/QuranRecitation.Data/Model/StudentDisciplineLevel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Web;

namespace QuranRecitation.Data.Model
{
    [Table("StudentDisciplineLevel")]
    public class StudentDisciplineLevel
    {
        [Key, Column(Order = 0)]
        public Guid StudentId { get; set; }
        public virtual Member Student { get; set; }

        [Key, Column(Order = 1)]
        public int DisciplineId { get; set; }
        public virtual Discipline Discipline { get; set; }


        public int DisciplineLevelId { get; set; }
        public virtual DisciplineLevel DisciplineLevel { get; set; }
    }
}

File: Madrasaty.Front/src/app/models/student-discipline-level.ts
import { Member } from './member'
import { Discipline } from './discipline'
import { DisciplineLevel } from './DisciplineLevel'

export class StudentDisciplineLevel {
    StudentId: string
    Student: Member
    DisciplineId: number
    Discipline: Discipline
    DisciplineLevelId: number
    DisciplineLevel: DisciplineLevel
}
